department-of-veterans-affairs/abd-vro
# MCP-2091: Add logic to determine claim evidence sufficiency. (Rhinitis and Sinusitis)

## What was the problem?

Logic to determine sufficient evidence did not make it in to the first round of PR's. 

Associated tickets or Slack threads:

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How does this fix it?[^1]

Adds logic to decide if a claim has enough evidence to be moved forward in fast track and tests for new data returned. 

More lighthouse test patients to test logic are needed. 

## How to test this PR
- Step 1 Check build an run unit tests, look at code. 
- Try ICN 9000682 for rhinitis. ICN 195 for sinusitis should qualify as having sufficient evidence (constant sinusitis)
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
